### DISCUSSION TITLE
How to prompt to not generate text/labels on images

### DISCUSSION THREAD

**Discussion by johannes-z** [lllyasviel/Fooocus#2913] [Q&A] [answered]
Hello! I'm using the `Mk Vintage Travel Poster` style to generate some images - I love the style but it adds non-sense text/labels to the images. I'd like to retain the style of the images, but not generate any text. I used the following as negative prompts, but they don't seem to have any effect. I guess that style adds texts to the prompt; is there any way I can extract the prompts and remove the ones that would add a text?

**Comment by mashb1t** [comment, marked as answer]
You can find the style files in the sdxl-styles folder and get both positive and negative prompt there. Starting from the next release 2.4.0, you can get the full positive/negative prompt also in the history log.
